pkgs/top-level/all-packages.nix

```diff
@@ -18421,6 +18421,7 @@ with pkgs;
 
   inherit (callPackage ../development/libraries/libdwarf { })
     libdwarf dwarfdump;
+  libdwarf_0_4 = callPackage ../development/libraries/libdwarf/0.4.nix { };
```

Comment:
If the packages are very similar I think using a common nix file would be better than copying it.

Reply:
I think the packages are not similar due to structure and build tools change